stable/metrics-server/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.3.1
 description: Metrics Server is a cluster-wide aggregator of resource usage data.
 name: metrics-server
-version: 2.1.1
+version: 2.1.2
```

Comment:
Needs a minor version bump since this adds functionality.

Reply:
fixed